graphql-dgs/src/main/kotlin/com/netflix/graphql/dgs/internal/DgsSchemaProvider.kt

```diff
@@ -241,13 +244,13 @@ class DgsSchemaProvider(private val applicationContext: ApplicationContext,
     private fun invokeDataFetcher(method: Method, dgsComponent: Any, environment: DataFetchingEnvironment): Any? {
         return try {
             val args = mutableListOf<Any?>()
-
-            method.parameters.forEach { parameter ->
+            val parameterNames = DefaultParameterNameDiscoverer().getParameterNames(method) ?: emptyArray()
+            method.parameters.forEachIndexed { idx, parameter ->
```

Comment:
the lambda is probably getting a bit large, should we reflect it as an internal method?

Reply:
Planning to address that separately